src/main/java/views/blackjack/blackjack.ftl.html

```diff
@@ -123,6 +123,41 @@
      });
 
 });
+
+//Function to handle hit requests from the user
+//Checks if the field of the hit button is "Hit"
+//If true then we make a post request to get a new card
+//Grab this new card, string configure it, and append
+//the new card to the users hand
+$("#dealHit").click(function(){
+    val = "";
+    data = "";
+    data = $("div.dealHit").text;
+
+    if(data == "Hit"){
+        var newCard;
+        //Send game data to get card{amount}
+        $.ajax({
+            type: "POST",
+            url: "/val/" + val,
+            data: JSON.stringify(game),
+            success: function(data, status){
+            console.log("Data: " + data + "\nStatus: " + status);
+            //Display game data
+            display(data);
+            game = data;},
+            contentType:"application/json; charset=utf-8",
+            dataType:"json",
+        });
+        val = game.getCard
```

Comment:
Currently there is no variable in the game, getCard. We can make one if you need for the hit. I think during the meeting Jacob was taking about displaying the most recently added card by using game.pHand[game.pHand.length] to get the last card that was added to the hand